src/nrfx/hal/nrf_timer.h

```diff
@@ -105,6 +164,80 @@ typedef enum
 void nrf_timer_task_trigger(NRF_TIMER_Type * p_reg,
                             nrf_timer_task_t task);
 
+/**
+ * @brief Function for clearing the specified timer event.
+ *
+ * @param[in] p_reg Pointer to the structure of registers of the peripheral.
+ * @param[in] event Event to clear.
+ */
+__STATIC_INLINE void nrf_timer_event_clear(NRF_TIMER_Type * p_reg, nrf_timer_event_t event);
```

Comment:
Where you can take a line directly from the original file, please do so (so there is no differences on comparison) (In these prototypes, the difference is that you placed the 2nd parameter in the same line, while in the original files, they are in the next line)